sdk-setup/src/mb2

```diff
@@ -157,8 +157,17 @@ GLOBAL OPTIONS
         Specify the device
 
     -o, --output-dir <dir>
-        Where to place produced RPM files. Defaults to ./RPMS directory.  With
-        this option existing files in the output directory are preserved
+        Defaults to './RPMS'. The resulting RPM packages will be placed
```

Comment:
I wonder if it could be stated a little bit more explicit that the order of options matter. "-o RPMS --no-search-output-dir" is not equal to "--no-search-output-dir -o RPMS".

Reply:
Done + a little more help polishing